src/schemas/HLR-create-request-body_v1/schema.json

```diff
@@ -0,0 +1,158 @@
+                    }
+                  },
+                  "additionalProperties": false,
+                  "required": [ "homeless" ]
+                },
+                "informalConferenceContact": { "type": "string", "enum": [ "veteran", "representative"] },
+                "informalConferenceTime":    { "type": "string", "enum": [ "800-1200 ET", "1200-1630 ET" ] },
+                "informalConferenceRep":     {
+                  "type": "object",
+                  "description" : "The Representative information listed MUST match the current Power of Attorney for the Veteran.  Any changes to the Power of Attorney must be submitted via a VA 21-22 form separately.",
+                  "properties": {
+                    "firstName": { "type": "string", "maxLength": 30 },
+                    "lastName":  { "type": "string", "maxLength": 40 },
+                    "phone": { "$ref": "#/definitions/hlrCreatePhone" },
+                    "email": {
+                      "type": "string",
+                      "format": "email",
+                      "minLength": 6,
+                      "maxLength": 255
+                    }
+                  },
+                  "additionalProperties": false,
+                  "required": [ "firstName", "lastName", "phone" ]
+                },
+                "socOptIn":                  { "type": "boolean" }
+              },
+              "required": [
+                "informalConference",
+                "benefitType",
+                "veteran",
+                "socOptIn"
+              ],
+              "if": { "properties": { "informalConference": { "const":  true } } },
+              "then": { "required":  ["informalConferenceContact", "informalConferenceTime"] }
```

Comment:
Draft-07 added explicit conditional schema evaluationI(if-then). Current version of Ajv JSON schema validator used by this repo for unit test does not support Draft-07